src/DynamicFormControl.vue

```diff
@@ -6,7 +6,7 @@
                   v-tooltip="formControl.helpText">
                 <help-circle-icon></help-circle-icon>
             </span>
-            <span v-if="formControl.required" class="small">(required)</span>
+            <span v-if="formControl.required" :class="{'text-danger': isRed(formControl.value), small: true}">(required)</span>
```

Comment:
I suspect the small: true syntax could be improved on

Reply:
You can actually just leave `class="small"` and also have `:class={...}` - Vue handily just appends the dynamic classes.